src/20_Components/amp-mustache.html

```diff
@@ -0,0 +1,111 @@
+<!--
+  ## Introduction
+
+  <a href="https://www.ampproject.org/docs/reference/components/amp-mustache">amp-mustache</a> templates are a simple, structured templating system.
+
+  amp-mustache doesn't actually provide the data, or compile the template.  Instead, the data is provided and template compiled by other AMP tags, such as amp-access, amp-form, and amp-list.  For more information on those use cases, see those docs/samples directly.
```

Comment:
nit: can you highlight component names using "`", e.g. `amp-mustache`.

Should we link to amp-access, amp-form, amp-list samples?